components/supervisor/pkg/supervisor/supervisor.go

```diff
@@ -544,11 +544,14 @@ func installDotfiles(ctx context.Context, cfg *Config, tokenService *InMemoryTok
 					Host: repoUrl.Host,
 					Kind: KindGit,
 				})
-				if err != nil {
-					return
+				if err == nil {
+					username = resp.User
+					password = resp.Token
+				} else {
+					username = "gitpod"
```

Comment:
i don't see the point. credential helper is using tokenService as well

Reply:
Does that mean `git` would still use `gp credential-helper` while performing the clone?